cmd/tracee-ebpf/printer.go

```diff
@@ -233,6 +233,7 @@ func (p *gobEventPrinter) Init() error {
 	p.outEnc = gob.NewEncoder(p.out)
 	gob.Register(external.Event{})
 	gob.Register(external.SlimCred{})
+	gob.Register(external.ProcessCtx{})
```

Comment:
Why do we need to register this struct?

Reply:
yes 
it's for the passing the first "snapshot of process tree to tracee rules

Reply:
i cant remove it and we  will pass it with the json printer , if u willing to let me know (:

Reply:
But I don't see such passing of the first snapshot.
If not part of this PR - let's add this in a future PR. Is that possible?

Reply:
yes sure!